core/src/test/java/io/grpc/internal/CensusModulesTest.java

```diff
@@ -1148,4 +1166,92 @@ private static void assertNoClientContent(StatsTestUtils.MetricsRecord record) {
     assertNull(record.getMetric(DeprecatedCensusConstants.RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES));
     assertNull(record.getMetric(DeprecatedCensusConstants.RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES));
   }
+
+  @SuppressWarnings("deprecation")
```

Comment:
Our convention is to annotate the test method as `@Derepcated`. This indicates these tests are to be deleted once the deprecated API under test is deleted.

Reply:
done